samples/oauth_php/VetsApiClaims/database/migrations/2019_09_13_170041_create_sessions_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateSessionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('oauth_sessions', function (Blueprint $table) {
```

Comment:
this should probably have an index - but, if this is just a demo, not really needed - however, i assume since this is public, someone will probably use it as a scaffold.